apps/scan/src/main/scala/org/lfdecentralizedtrust/splice/scan/admin/api/client/BftScanConnection.scala

```diff
@@ -125,6 +125,27 @@ class BftScanConnection(
   private val refreshAction: Option[PeriodicAction] = scanList match {
     case _: BftScanConnection.TrustSingle =>
       None
+    case ts: BftScanConnection.TrustSpecificScanList =>
+      Some(
+        new PeriodicAction(
+          clock,
+          com.digitalasset.canton.time.NonNegativeFiniteDuration
+            .fromConfig(ts.scansRefreshInterval),
+          loggerFactory,
+          retryProvider.timeouts,
+          "refresh_scan_list_trust_specific",
+        )({ tc =>
+          FutureUnlessShutdown.outcomeF(
+            retryProvider.retry(
+              RetryFor.LongRunningAutomation,
+              "refresh_trust_specific_scan_list",
+              "refresh_trust_specific_scan_list",
```

Comment:
One of these two  is the description, i suggest we include the `ts` there for debugging purposes.